vm/src/objlist.rs

```diff
@@ -9,6 +9,36 @@ fn get_pos(l: &Vec<PyObjectRef>, p: i32) -> usize {
     }
 }
 
+pub fn get_slice_items(l: &Vec<PyObjectRef>, slice: &PyObjectRef) -> Vec<PyObjectRef> {
+    // TODO: we could potentially avoid this copy and use slice
+    match &(slice.borrow()).kind {
+        PyObjectKind::Slice { start, stop, step } => {
```

Comment:
Should we use here rust's `if let` contraption? Since we only have two arms?

Reply:
I think we have the same problem as before; because we want to capture the value of the default for the error message.

Reply:
Ah, I see, yes you are right.